pymc/sampling.py

```diff
@@ -2093,6 +2094,73 @@ def sample_prior_predictive(
     return pm.to_inference_data(prior=prior, **ikwargs)
 
 
+def draw(
+    vars: Union[Variable, Sequence[Variable]],
+    draws=1,
+    mode: Optional[Union[str, Mode]] = None,
+    **kwargs,
+) -> List[np.ndarray]:
+    """Draw samples for one variable or a list of variables
+
+    Parameters
+    ----------
+    vars : Union[Variable, Sequence[Variable]]
```

Comment:
You shouldn't use type hints in the docstring but numpydoc formatting: https://numpydoc.readthedocs.io/en/latest/format.html#parameters